lib/serialize.js

```diff
@@ -56,78 +58,79 @@ export async function pageProps({ params }) {
               },
-              children: [
-                {
-                  type: 'element',
-                  tagName: 'path',
-                  properties: {
-                    strokeLinecap: 'round',
-                    strokeLinejoin: 'round',
-                    strokeWidth: 2,
-                    d: 'M13.828 10.172a4 4 0 00-5.656 0l-4 4a4 4 0 105.656 5.656l1.102-1.101m-.758-4.899a4 4 0 005.656 0l4-4a4 4 0 00-5.656-5.656l-1.1 1.1'
-                  },
-                  children: []
-                }
-              ]
+              content: {
+                type: 'element',
+                tagName: 'svg',
+                properties: {
+                  className: ['h-6', 'w-6', 'ml-2', 'docs-copy-btn'],
+                  xmlns: 'http://www.w3.org/2000/svg',
+                  fill: 'none',
+                  viewBox: '0 0 24 24',
+                  stroke: 'currentColor'
+                },
+                children: [
+                  {
+                    type: 'element',
+                    tagName: 'path',
+                    properties: {
+                      strokeLinecap: 'round',
+                      strokeLinejoin: 'round',
+                      strokeWidth: 2,
+                      d: 'M13.828 10.172a4 4 0 00-5.656 0l-4 4a4 4 0 105.656 5.656l1.102-1.101m-.758-4.899a4 4 0 005.656 0l4-4a4 4 0 00-5.656-5.656l-1.1 1.1'
+                    },
+                    children: []
+                  }
+                ]
+              }
             }
-          }
-        ]
-      ],
-      remarkPlugins: [
-        remarkHeadingId,
-        remarkGfm,
-        [
-          remarkCodeImport,
-          {
-            disabled: process.env.DOCS_REPO ? true : false, // only works with local filesystem, not remote fetch
-            importBasePath: importBasePath
-          }
+          ]
         ],
-        [remarkRewriteImages, { destination: process.env.ASSETS_DESTINATION }],
-        remarkInlineLinks,
-        [remarkExternalLinks, { target: false, rel: ['nofollow'] }],
-        [
-          remarkInternalLinks,
-          {
-            prefix: docsFolder,
-            slug: inlineLinkSlugHelper,
-            extensions: ['.mdx', '.md'],
-            trailingSlash: trailingSlash
-          }
+        remarkPlugins: [
+          remarkHeadingId,
+          remarkGfm,
+          remarkCodeImport,
+          [
+            remarkRewriteImages,
+            { destination: process.env.ASSETS_DESTINATION }
+          ],
+          remarkInlineLinks,
+          [remarkExternalLinks, { target: false, rel: ['nofollow'] }],
+          [
+            remarkInternalLinks,
+            {
+              prefix: docsFolder,
+              slug: inlineLinkSlugHelper,
+              extensions: ['.mdx', '.md'],
+              trailingSlash: trailingSlash
+            }
+          ]
         ]
-      ]
-    },
-    target: ['esnext']
-  })
+      },
+      target: ['esnext']
+    }
+  )
```

Comment:
The rest of this file has been reformatted by prettier according to the configuration in `.prettierrc` in the repo root.